src/Gateway.php

```diff
@@ -0,0 +1,305 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Exan\Fenrir;
+
+use Exan\Fenrir\Bitwise\Bitwise;
+use Exan\Fenrir\Const\Events as Events;
+use Exan\Fenrir\Const\WebsocketEvents;
+use Exan\Fenrir\Enums\Gateway\StatusType;
+use Exan\Fenrir\Websocket\Helpers\ActivityBuilder;
+use Exan\Fenrir\Websocket\Objects\D\Hello;
+use Exan\Fenrir\Websocket\Objects\Payload;
+use JsonMapper;
+use Psr\Log\LoggerInterface;
+use Psr\Log\NullLogger;
+use Ratchet\RFC6455\Messaging\MessageInterface;
+use React\EventLoop\LoopInterface;
+use React\EventLoop\TimerInterface;
+
+class Gateway
+{
+    public EventHandler $events;
+
+    public const WEBSOCKET_URL = 'wss://gateway.discord.gg/?v=10';
```

Comment:
```suggestion
    public const WEBSOCKET_URL = 'wss://gateway.discord.gg/?v=10';

    public EventHandler $events;
```
Constants before properties